examples/dinput8_hook/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+#![feature(once_cell)]
+#![allow(non_snake_case, unused_variables, unreachable_patterns)]
```

Comment:
I'd strongly prefer not to use these. These disabled lints are some of those I deem most important for maintaining high code quality.